tools/build/package.json

```diff
@@ -52,6 +52,7 @@
     "rollup-plugin-polyfill-node": "0.12.0"
   },
   "peerDependencies": {
+    "@babel/core": "7.21.8",
```

Comment:
What should the changeset be for this?

Reply:
Just a patch. (it's v0, so it doesn't really matter)